build.sh

```diff
@@ -71,7 +82,18 @@ else
 	popd
 fi
 
-if [ "$ARCH" = "arm" ]; then
+if [[ $TARGET_OS == 'android' ]]; then
+	echo "Setting gclient target_os to android"
+	echo "target_os = [ 'android' ]" >> $WEBRTC_DIR/.gclient
+	gclient config --name src $WEBRTC_REPO || exit 1
+	gclient sync --with_branch_heads -r $COMMIT || exit 1
```

Comment:
This seems redundant with L81

Could you just add an `if` block there for appending the `target_os`?

Reply:
It needs to run the whole config and sync again to get all the Android deps:  https://chromium.googlesource.com/chromium/src/+/master/docs/android_build_instructions.md

The `gclient` stuff for Android is a mystery to me.  I got it working once manually in a VM, but then have never been able to repeat it.  I've tried many many variations.  At this point, I'm out of time.  But this is very close to what I got working in the VM, and I'm out of time for now.